docs/learn/learn-polkadot-opengov.md

```diff
@@ -530,6 +530,24 @@ Note that to successfully cast votes you need to have the
 [existential deposit](./learn-accounts.md#existential-deposit-and-reaping) and some additional funds
 to pay for transaction fees.
 
+### Voting Locks
+
+A lock is placed on the {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} when you vote on an
+Ongoing referendum. If you choose to remove your vote when the referendum is still in the decision period, 
+your voting lock is removed. Once the referendum is decided (accepted or rejected), your {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} 
+will be available for unlocking if you voted with zero conviction. The governance app or interface you 
+used for participating in Polkadot OpenGov should show an option to unlock your {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}.
```

Comment:
There is actually a section on the Wiki

https://wiki.polkadot.network/docs/learn-guides-accounts#unlocking-locks

Reply:
Yeah, but it is generic to all kinds of locks and it is on one of those Polkadot-JS pages. Redundancy here explaining specifically on voting locks is okay and handy to share.